scss/partials/_menu.scss

```diff
@@ -37,7 +37,7 @@
   &__logo {
     position: absolute;
     right: 2.6rem;
-    z-index: 10;
+    z-index: 101;
```

Comment:
Just thinking out loud that we should probably think about exactly how we're incrementing z-indexes on elements so that we're not just picking numbers out of a hat in hopes that they are higher or lower than the other things we're trying to layer around.

Reply:
I agree. 
The reason I went with this value is that the `.navbar` has a z-index of 100, so I incremented the value by one. 
But you are correct that we have been using somewhat arbitrary values and may benefit from having a conversation about this.